criu/crtools.c

```diff
@@ -422,6 +422,8 @@ int main(int argc, char *argv[], char *envp[])
 	       "                        can be 'nftables' or 'iptables' (default).\n"
 	       "  --unprivileged        accept limitations when running as non-root\n"
 	       "                        consult documentation for further details\n"
+	       "  --parallel            enable parallel restore of AMDGPU buffer object content\n"
+	       "                        with other restore operations\n"
```

Comment:
@wweewrwer Is there any reason why parallel restore of buffer objects should not be enabled by default?

Reply:
To the best of my ability, I can't think of any scenarios where parallel restore wouldn't be applicable. I think the default method is simpler to implement, while the parallel method is more efficient. The state of processes restored through parallel restore has no differences from those restored using the default method. If most of the criu team prefer to enable the optimization by default, we can make the change :)

Reply:
I tested this functionality with Podman, but it causes container restore to hang:
```
(00.000986) Forking task with 1 pid (flags 0x6c028000)
(00.000987) Creating process using clone3()
(00.001105) PID: real 389586 virt 1
(00.001120) Run restore asynchronous hook from criu master for external devices
(00.001121) plugin: `amdgpu_plugin' hook 12 -> 0x7fcb95a16f83
(00.001122) amdgpu_plugin: Begin to recv parallel restore cmd
(00.001132) Parallel restore: begin to recv cmd_head
(00.001140)      1: Found id extRootNetNS (fd 14) in inherit fd list
(00.001219)      1: timens: monotonic -602 193779164
(00.001223)      1: timens: boottime -602 193774845
```
Is it intended to work with containers?

Reply:
It can be enabled by default.

Reply:
@avagin Thanks for the confirmation! We will make the change.